src/search_cuda.cpp

```diff
@@ -125,9 +133,30 @@ void BeamSearch_Cuda::SelectTop() {
   std::span<int32_t> next_indices{topk_next_indices_.get(), size};
 
 #if 0
-  DumpCudaMemory("Next Scores", next_scores);
-  DumpCudaMemory("Next Tokens", next_tokens);
-  DumpCudaMemory("Next Indices", next_indices);
+  // Copy next_tokens, next_indices, next_scores to CPU
```

Comment:
replace it with DumpCudaSpan: https://github.com/microsoft/onnxruntime-genai/blob/00ceb80e1984c408459dfabe92a5b4eb97318578/src/models/debugging.h#L15